lib/puppet/defaults.rb

```diff
@@ -154,6 +154,19 @@ def self.default_vendormoduledir
         end
       end
     },
+    :merge_dependency_warnings => {
+      :default => false,
+      :type    => :boolean,
+      :desc    => "Whether to merge class-level dependency failure warnings.
+
+        When a class has a failed dependency, every resource in the class
+        generates a notice level message about the dependency failure,
+        and a warning level message about skipping the resource.
+
+        If true, all messages caused by a class dependency failure are coalesced
```

Comment:
`are merged`?

Reply:
Done! Can we alt least keep `coalesce` in the commit message 🙃